Explorer/Assets/DCL/Multiplayer/Movement/Settings/Rules/VelocitySpeedTiersRule.cs

```diff
@@ -0,0 +1,45 @@
+﻿using DCL.CharacterMotion.Components;
+using UnityEngine;
+
+namespace DCL.Multiplayer.Movement.Settings
+{
+    public class VelocitySpeedTiersRuleBase : SendRuleBase
+    {
+        [Header("SPEED SENT RATES")]
+        public float SprintSentRate;
+        public float RunSentRate;
+        public float WalkSentRate;
+
+        [Header("SPEED THRESHOLDS")]
+        public float SprintSqrSpeed;
```

Comment:
I think these tiers should come from other settings, otherwise they are duplicated here, WDYT?